django/website/templates/website/FAQ.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'site_base_template.html' %} 
+{% extends 'site_base_template.html' %}
```

Comment:
My editor noted a few trailing whitespaces, so I removed those as I crafted the the FAQs.